api/services/userService.js

```diff
@@ -72,10 +72,10 @@ module.exports = {
         if (points >= 10) return true;
         break;
       case 3:
-        if (points >= 7) return true;
+        if (points >= 5) return true;
```

Comment:
While we are at it can we rewrite all these switch cases? It's a bad practice to use a conditional when we just want to return the boolean expression
```suggestion
        return points >= 5;
```

Can we do this kind of move for all 4 cases?